packages/gestalt/src/ComboBox.js

```diff
@@ -11,59 +11,132 @@ import {
   useMemo,
   useRef,
   useState,
+  type Ref,
 } from 'react';
 import Box from './Box.js';
 import Layer from './Layer.js';
 import Popover from './Popover.js';
 import Text from './Text.js';
 import InternalTextField from './InternalTextField.js';
 import Tag from './Tag.js';
-import ComboBoxItem, { type ComboBoxItemType } from './ComboBoxItem.js';
+import ComboBoxItem from './ComboBoxItem.js';
 import { ESCAPE, TAB, ENTER, UP_ARROW, DOWN_ARROW } from './keyCodes.js';
 import handleContainerScrolling, {
   KEYS,
   type DirectionOptionType,
 } from './utils/keyboardNavigation.js';
 
 type Size = 'md' | 'lg';
+type OptionType = {|
+  label: string,
+  subtext?: string,
+  value: string,
+|};
 
 type Props = {|
```

Comment:
Do you mind alphabetizing these props while you're in here, please?

Reply:
DONE